nautobot/docs/administration/nautobot-server.md

```diff
@@ -127,6 +127,11 @@ nautobot=> \q
 !!! note
     This is a built-in Django command. Please see the [official documentation on `dbshell`](https://docs.djangoproject.com/en/stable/ref/django-admin/#dbshell) for more information.
 
+### `dumpdata`
+
```

Comment:
Please add example syntax of this command and documentation of its common/relevant arguments, similar to the other entries in this file.